### PR TITLE
Allows admins to save their CC characters

### PR DESCRIPTION
**Pull request by Kyep**
Goals:
1) Allow admins to easily save their CC characters from one round to another, rather than having to re-create them every time via spawning items and modifying mob variables.
2) Make it easier for admins to quickly create a character for on-station responses to faxes, leading ERTs, etc.

Changes:
- Adds two admin-only jobs, "Nanotrasen Navy Officer", and "Special Operations Officer". The former is for IC investigations by admins of situations on the Cyberiad, and spawns in the admin room. The latter is for leading/overwatching ERT teams, and spawns at the ERT director's office. These existed previously, but they are now proper jobs that admins can join the round as via the latejoin screen.
- Neither of these jobs are accessible to normal players, either on join, in preferences, or as a random job. They're invisible to players.
- When they are spawned, no "X has arrived" announcement is made, and they do not show
up on the crew manifest either. They do, however, get PDAs. 
- The "select equipment" debug verb has also had relevant outfits adjusted slightly for consistency.
- The teleporter on admin station actually works now, and admins/NTNOs can use it to teleport directly to bridge.

Yes, I realize that admins could previously save their CC characters, and either:
A) join as a normal job, then change to their CC job
or
B) have everything written down, spawn a mob, and edit it to match their notes

That said, (A) leaves manifest oddities, and (B) is time consuming. This PR enables admins to do it much more effectively - to have a certain char slot they only use for joining as CC officers, enabling them to quickly jump into their CC character without having to re-create it every time, or worry about messing up the manifest.

:cl: Kyep
rscadd: Admins can now join as "Nanotrasen Navy Officer", and "Special Operations Officer". They spawn on adminstation and ERT office, respectively. They are not announced and do not appear on crew manifest.
tweak: "Select equipment" verb outfits tweaked for consistency with the above job outfits
fix: Teleporter on admin station now works.
/:cl:

### CODE REVIEW

**Comment by Necaladun** [comment]
...useful. Manifest oddities are annoying too, and it saves time. I like

**Comment by Earthdivine** [comment]
On one hand, didn't we have this once and it got taken out for something? Like "we don't need this space for the admins to keep their special characters" or what not.

On the other hand, this was a big complaint when it came to trying to do fax or CC related events, so :+1:

**Comment by KasparoVy** [comment]
I think it'd be super useful. It's just one of those features I feel's nice to have, and makes no sense to deny. Cool stuff :+1:

**Comment by Kyep** [comment]
@TheDZD Merge conflict resolved.

**Comment by Kyep** [comment]
In the last patch, I have:
- Created a new type of death alarm implant that dusts its owner when they die. It also performs the same alarm function as a death alarm - and is immune to EMP.
- Made these standard implants for NT Navy Officers and Special Operations Officers.

Rationale:
- Makes it less likely that a CC character's gear will be looted by normal crew in the event that they die. Still possible in theory if they're gibbed, but should be less likely now.
- Makes it more likely that a CC character's death will be noticed, since they have death alarms as standard.

File: code/game/objects/items/weapons/implants/implant_death_alarm.dm [hunk 1]
```diff
@@ -68,4 +68,42 @@
...
\ No newline at end of file
+	return 0
+
+
+
+/obj/item/weapon/implant/death_alarm/dust
+	name = "duster implant"
+	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+
+/obj/item/weapon/implant/death_alarm/dust/activate(var/cause)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/implants/implant_death_alarm.dm]: This whole thing is a huge amount of copy-pasta.

If you want to make it immune to EMPs, then make its `emp_act` return.

If you want it to dust on death, then add a var to the base alarm, that, if set, dusts the person on death (there's a proc for this too, by the by: `dust`) when it legitimately activates.

Then you won't have literally 90% of the death alarm code copy-pasted.

**Comment by Kyep** [comment]
Changes:
- CC officers now spawn with an internals box in their backpacks, like everyone else.
- CC officers now get dust and death alarm implanters in their backpacks on spawn, rather than  automatically having the implants in their bodies. This enables admins to test things on admin station without worrying about dusting or having their deaths broadcast.
- Dust implants no longer provide death alarm functionality.
- Dust implants no longer process().
- Dust implants are now based on explosive implants, and can be manually activated.
- CC officers now spawn with CC headsets that are named 'centcom headset'.

File: code/game/objects/items/weapons/implants/implant_death_alarm.dm (hunk 1 above)
Comment (Kyep) [inline code/game/objects/items/weapons/implants/implant_death_alarm.dm]: Commit I just did rewrote dust alarm implants to be more based on explosive implants, and remove the duplicate death alarm functionality entirely.

See what you think of it now.

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -133,6 +133,10 @@
 	name = "Nanotrasen ERT Radio Encryption Key"
 	channels = list("Response Team" = 1, "Science" = 1, "Command" = 1, "Medical" = 1, "Engineering" = 1, "Security" = 1, "Supply" = 1, "Service" = 1)
 
+/obj/item/device/encryptionkey/centcom
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/radio/encryptionkey.dm]: This is redundant when it's the same key as above.

File: code/game/objects/items/weapons/implants/implant_explosive.dm [hunk 3]
```diff
@@ -125,4 +125,49 @@
...
+		activate("death")
+
+/obj/item/weapon/implant/dust/activate(cause)
+	if(!cause || !imp_in || cause == "emp")
+		return 0
+	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your dusting implant? This will turn you to ash!", "Dusting Confirmation", "Yes", "No") != "Yes")
+		return 0
+	to_chat(imp_in, "<span class='notice'>Your dusting implant activates!</span>")
+	imp_in.visible_message("<span class = 'warning'>[imp_in] burns up in a flash!</span>")
+	var/obj/effect/decal/remains/human/R = new /obj/effect/decal/remains/human(imp_in.loc)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/implants/implant_explosive.dm]: this should use `imp_in.dust()` instead. Then it'll call all the thing in `death` and such before spawning a skeleton and stuff.

**Comment by Fox-McCloud** [comment]
Merge conflicts....

**Comment by Kyep** [comment]
@Fox-McCloud map merge conflict resolved.

**Pull request merged**